packages/blockchain-wallet-v4-frontend/src/modals/BuySell/LinkedPaymentAccounts/Accounts/ApplePay/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { Content, DisplayContainer, DisplayIcon, DisplayTitle } from 'components/BuySell'
+
+const DisplayContainerApplePay = styled(DisplayContainer)`
+  height: 74px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const RawApplePayIcon = (props) => (
```

Comment:
Can you put this svg into `blockchain-info-components/src/images/img` dir?

Reply:
gonna do